django_event_sourcing/registers.py

```diff
@@ -17,6 +17,11 @@ def __init__(self, event_type_classes):
                 self.data[event_type.fully_qualified_value] = event_type
 
 
+RegisteredSideEffect = collections.namedtuple(
+    "RegisteredSideEffect", field_names=("side_effect", "condition")
```

Comment:
```suggestion
    "RegisteredSideEffect", field_names=("callable", "condition")
```

I wonder if this is better since the it's currently `side_effect.side_effect` which might be confusing.